<!DOCTYPE html>
<html><head><title>UnauthorizedException</title><link rel="stylesheet" href="../../styles.css"/><script src="../../scripts.js"></script></head><body onload="ro();">
<div class="rH">2 instantiations of UnauthorizedException</div><div class="rA">Microsoft.Azure.ServiceBus (2)</div><div class="rG" id="Microsoft.Azure.ServiceBus"><div class="rF"><div class="rN">Amqp\AmqpExceptionHelper.cs (1)</div>
<a href="../Amqp/AmqpExceptionHelper.cs.html#122"><b>122</b>return new <i>UnauthorizedException</i>(message);</a>
</div>
<div class="rF"><div class="rN">Management\ManagementClient.cs (1)</div>
<a href="../Management/ManagementClient.cs.html#1010"><b>1010</b>return new <i>UnauthorizedException</i>(exceptionMessage);</a>
</div>
</div>
<div class="rH">2 references to UnauthorizedException</div><div class="rA">Microsoft.Azure.ServiceBus.Tests (2)</div><div class="rG" id="Microsoft.Azure.ServiceBus.Tests"><div class="rF"><div class="rN">SenderReceiverTests.cs (2)</div>
<a href="../../Microsoft.Azure.ServiceBus.Tests/SenderReceiverTests.cs.html#412"><b>412</b>await Assert.ThrowsAsync&lt;<i>UnauthorizedException</i>&gt;(</a>
<a href="../../Microsoft.Azure.ServiceBus.Tests/SenderReceiverTests.cs.html#416"><b>416</b>await Assert.ThrowsAsync&lt;<i>UnauthorizedException</i>&gt;(</a>
</div>
</div>
</body></html>